broker/cloud_run/lsst/classify_upsilon/main.py

```diff
@@ -62,78 +60,82 @@ def run() -> tuple[str, int]:
+                },
+                schema_name="default",
+            )
+        )
 
+    return "", HTTP_204
 
-def _classify_with_upsilon(alert_lite: pittgoogle.Alert) -> dict:
-    # extract the alert
-    alert_lite_dict = alert_lite.dict["alert_lite"]
-    alert_df = _create_dataframe(alert_lite_dict)
-    bands = ["u", "g", "r", "i", "z", "y"]
-    outgoing_dict = {
-        "diaObjectId": alert_lite_dict["diaObject"]["diaObjectId"],
-        "diaSourceId": alert_lite_dict["diaSource"]["diaSourceId"],
-    }
 
-    for band in bands:
-        # define parameters for feature extractions
-        filter_diaSources = alert_df[alert_df["band"] == band]
+def _classify_with_upsilon(alert_lite_df: pd.DataFrame) -> dict:
+    upsilon_dict = {}
+    rf_model = upsilon.load_rf_model()  # load UPSILoN's classification model
+    for band in survey_bands:
+        # ---Extract data
+        filter_diaSources = alert_lite_df[alert_lite_df["band"] == band]
         flux_gt_zero = filter_diaSources["psfFlux"].to_numpy() > 0
-
-        # set output to None if data is absent or there are too few data points for this band; limit set by UPSILoN
-        if filter_diaSources.empty or flux_gt_zero.sum() < 7:
-            outgoing_dict[f"{band}_label"] = None
-            outgoing_dict[f"{band}_probability"] = None
-            outgoing_dict[f"{band}_flag"] = None
+        # set output to None if data is absent or there are too few data points for this band
+        # limit recommended by UPSILoN
+        if filter_diaSources.empty or flux_gt_zero.sum() <= 80:
+            upsilon_dict[f"{band}_label"] = None
+            upsilon_dict[f"{band}_probability"] = None
+            upsilon_dict[f"{band}_flag"] = None
             continue
 
+        # ---Extract features
         flux = filter_diaSources["psfFlux"].to_numpy()[flux_gt_zero]
         flux_err = filter_diaSources["psfFluxErr"].to_numpy()[flux_gt_zero]
-
-        # UPSILoN requires three features to make a prediction, define them below:
         date = filter_diaSources["midpointMjdTai"].to_numpy()[flux_gt_zero]
         mag = _convert_flux_to_mag(flux)
         mag_err = _calculate_mag_err(flux, flux_err)
-
-        # extract features
         e_features = upsilon.ExtractFeatures(date, mag, mag_err)
         e_features.run()
         features = e_features.get_features()
 
-        # classify
+        # ---Classify
         label, probability, flag = upsilon.predict(rf_model, features)
-        outgoing_dict[f"{band}_label"] = label
-        outgoing_dict[f"{band}_probability"] = probability
-        outgoing_dict[f"{band}_flag"] = flag
+        upsilon_dict[f"{band}_label"] = label
+        upsilon_dict[f"{band}_probability"] = probability
+        upsilon_dict[f"{band}_flag"] = flag
 
-    return outgoing_dict
+    return upsilon_dict
 
 
-def _create_dataframe(alert_dict: pittgoogle.Alert) -> pd.DataFrame:
+def _create_lite_dataframe(alert_dict: pittgoogle.Alert) -> pd.DataFrame:
```

Comment:
```suggestion
def _create_lite_dataframe(alert_dict: dict) -> pd.DataFrame:
```

Right? If not, the keys being used below in `alert_dict.get()` are wrong.